### DISCUSSION TITLE
AutoTagger?

### DISCUSSION THREAD

**Discussion by danakaufhold** [AprilSylph/XKit-Rewritten#269] [Suggestion Box]
Hi! First, I wanted to say thank you so very much for being the next in a long line of folks to take up the XKit mantle. I'm so grateful that you've decided to take on this responsibility for all of us who rely on XKit to fix the Hellsite (tm).

That being said, I do have one ask/request, that you consider re-enabling AutoTagger as one of your XKit apps! I (and many other users) used it to add tags easily based on if posts were queued, the type of post, replies, etc. and it went very well hand-in-hand with Quick Reblogs.

I know you folks are probably incredibly busy cleaning up Tumblr Staff's messes (lol), but I thought I'd throw my suggestion in the ring nonetheless. Thank you again for all your hard work!

**Comment by AprilSylph** [comment]
Please take a glance at existing discussions before opening your own. See #225 for your answer.
